src/docparser.cpp

```diff
@@ -5567,8 +5567,30 @@ int DocPara::handleCommand(const QCString &cmdName, const int tok)
         defaultHandleTitleAndSize(CMD_STARTUML,dv,dv->children(),width,height);
         doctokenizerYYsetStatePlantUML();
         retval = doctokenizerYYlex();
-        int line=0;
-        dv->setText(stripLeadingAndTrailingEmptyLines(g_parserContext.token->verb,line));
+        int line = 0;
+        QCString trimmedVerb = stripLeadingAndTrailingEmptyLines(g_parserContext.token->verb,line);
+        if (engine == "ditaa")
+        {
+          dv->setUseBitmap(true);
+        }
+        else if (engine == "uml")
+        {
+           // check on ditaa in first line
+           if (!trimmedVerb.isEmpty())
+           {
+             const char *p = trimmedVerb.data();
```

Comment:
this loop seems like a difficult way to do `int i = trimmedVerb.find('\n')`

Reply:
indeed probably will be easier to use find, I'll have a look.